### DISCUSSION TITLE
02UM frame drops during playback

### DISCUSSION THREAD

**Discussion by ArooLee** [nirgoren/Eddienput#7] [Technical support]
Using the kof_keyboard.json config file I have set up a simple recording that inputs qcf+A, low blocks for 120f, releases low block, and then repeats that sequence 2 additional times. During the playback of that (and any other sequence), frames are being dropped in the game. I've recorded a video and posted it below. There are some instances where you can hear my stick inputs but they aren't registered by the game as they are inputted during a particularly bad hangup.

I've tried the following:

- Using kof.json instead of kof_keyboard
- Unplugging my stick and using keyboard for P1 controls
- Different configurations of the things mentioned above
- Increasing 02UM's priority in the task manager

I've tested the bot in GG+R and it performs as expected without any hiccups. Another user rylander is experiencing the same issues as I've described above with 02UM.

https://user-images.githubusercontent.com/10816145/113931289-24f06900-97a7-11eb-9f22-f72893849942.mp4

**Comment by DesChamos** [comment]
I doubt it’ll come down to them, but it may be helpful if you post your specs. I don’t have 02UM so I can’t test this on my setup.

**Comment by nirgoren** [comment]
A change introduced in v1.1 may help with this.
